tests/epel-centos7/Dockerfile

```diff
@@ -16,6 +16,6 @@ RUN yum clean all && \
 
 RUN mkdir /etc/ansible/
 RUN echo -e '[local]\nlocalhost' > /etc/ansible/hosts
-RUN yum -y install ansible
+RUN yum -y install ansible ansible-lint
```

Comment:
When I run ansible-lint similar (I hope) to what the test does I get errors:

<pre>
$ ansible-lint */*.yml 
[ANSIBLE0012] Commands should not change things if nothing needs doing
$HOME/Git/ansible-role-stress/tasks/main.yml:38
Task/Handler: run stress-ng
$ ansible-lint --version
ansible-lint 3.4.21
</pre>

But I don't see these in the travis job out. I expect that is because command doesn't find *.yml files or something. One can run ansible-lint with -v for some more debug output.

Reply:
I think you are running ansible-lint against the wrong branch.
`$ git status
On branch FIX-20180530
Your branch is up to date with 'origin/FIX-20180530'.

nothing to commit, working tree clean`

Reply:
You are correct. Could you squash all the commits into one commit?

Reply:
Sure.